csharp/src/Google.Protobuf/JsonFormatter.cs

```diff
@@ -156,17 +157,22 @@ public string Format(IMessage message)
         /// <param name="writer">The TextWriter to write the formatted message to.</param>
         /// <returns>The formatted message.</returns>
         public void Format(IMessage message, TextWriter writer)
+        {
```

Comment:
Sure. I tend to apply the same approach to code improvements. I'll use modern style for any new code (like `var foo =...` instead of `string foo = x.tostring()`